.gitignore

```diff
@@ -1 +1,2 @@
 __pycache__/
+myenv/
```

Comment:
correct! 

please consider next time to name your virtualenvironment with its commono name 'venv'
follow best practice, conventions and so on..

i dislike myenv